pkg/config/setup/system_probe.go

```diff
@@ -404,6 +404,7 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Config) {
 	cfg.BindEnvAndSetDefault(join(discoveryNS, "enabled"), false)
 	cfg.BindEnvAndSetDefault(join(discoveryNS, "cpu_usage_update_delay"), "60s")
 	cfg.BindEnvAndSetDefault(join(discoveryNS, "ignored_command_names"), []string{"chronyd", "cilium-agent", "containerd", "dhclient", "dockerd", "kubelet", "livenessprobe", "local-volume-pr", "sshd", "systemd"})
+	cfg.BindEnvAndSetDefault(join(discoveryNS, "ignored_services"), []string{"datadog-agent", "trace-agent", "process-agent", "datadog-cluster-agent"})
```

Comment:
Add `security-agent` and other processes listed in `cmd/*`?

Reply:
APM Distributed Tracing team provided us with some test environment with service discovery enabled, I took some DD services from that environment. Apparently security is not enabled there, that's why I don't see that service-agent in the discovery list. I would be careful about adding names that I don't know how they show up in discovery. This is why I prefer not to extend this list at this point.

Reply:
@yuri-lipnesh  - Bryce is mentioning another DD product that it is not enabled in your environment. Please open a follow up PR to address that, and run PR in dogfooding.

Reply:
I've created [USMON-1339](https://datadoghq.atlassian.net/browse/USMON-1339)